assets/js/common/DismissableToast/DismissableToast.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { toast } from 'react-hot-toast';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+import DismissableToast from '.';
+
+describe('DismissableToast component', () => {
+  afterEach(() => {
+    // restore the spy created with spyOn
```

Comment:
Just a general question.
Leaving comments in tests is fine ? 
Some tests are quite complex, I find it useful :+1:

Reply:
This test is not complex but yeah I think we don't use spyOn this way so often, so i left for the future trento devs :D